### PR TITLE
Assignment 3

### PR DESCRIPTION
**Pull request by jacob-berger**
Implemented all basic requirements.

### CODE REVIEW

File: Inheritance/test/Inheritance.Tests/PrinterTests.cs [hunk 1]
```diff
@@ -34,9 +34,47 @@ public void PersonGetsPrinted()
                 }
             }
```
Comment (jake-mcg) [inline Inheritance/test/Inheritance.Tests/PrinterTests.cs]: I didn't see any unit tests for food or television items. I tested the PrintInfo methods separately in their own test classes (FoodTests and TelevisionTests), and then I tested that Item can be set to a Television or Food object respectively. Also, good call on testing for null objects

File: Inheritance/src/Inheritance/ActorExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            //} else if (actor.GetType() == typeof(Penny))
+            //{
+            //    Penny temp = new Penny();
+            //    temp.Speak();
+            //} else
+            //{
+            //    Sheldon temp = new Sheldon();
+            //    temp.Speak();
+            //}
+
```
Comment (matHopg) [inline Inheritance/src/Inheritance/ActorExtensions.cs]: This is a very elegant solution to the normally cumbersome switch statements. This at least taught me some useful tricks within c#.
